### PR TITLE
PORT-4628-terraform-add-examples-of-webhook-and-scorecard-resources

### PR DESCRIPTION
**Pull request by dvirsegev**

### CODE REVIEW

File: examples/resources/port_scorecard/main.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  rules = [{
+    identifier = "high-avalability"
+    title      = "High Availability"
+    level      = "Gold"
+    query = {
+      combinator = "and"
+      conditions = [{
+        property = "replicaCount"
+        operator = ">="
+        "value"  = "4"
```
Comment (MPTG94) [inline examples/resources/port_scorecard/main.tf:14]: Should value have quotes? should the number have quotes?

File: examples/resources/port_webhook/main.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
+resource "port_webhook" "github" {
+  identifier = "github"
+  title      = "Github"
+  icon       = "Terraform"
```
Comment (MPTG94) [inline examples/resources/port_webhook/main.tf:4]: Why not icon github?

**Review by MPTG94 — APPROVED** [review]
LGTM

File: examples/resources/port_webhook/main.tf (hunk 2 above)
Comment (dvirsegev) [reply examples/resources/port_webhook/main.tf:4]: That depends, but I think we want to see in the ingest modal an icon of terraform and not Github ?

File: examples/resources/port_scorecard/main.tf (hunk 1 above)
Comment (dvirsegev) [reply examples/resources/port_scorecard/main.tf:14]: Currently, In terraform I only support value of string, I need to add support of  number value  and boolean value

**Pull request merged**